CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.2.2 - 2022-07-12
+
+### Added
```

Comment:
An `Added` should never be in a patch release as that goes against semver... it needs to be in either a minor or major release, so you'll need to bump the version to 0.3.0 or similar.